build.zig

```diff
@@ -17,10 +17,54 @@ pub fn build(b: *std.Build) !void {
         },
     };
 
+    const c_sdl_preferred_linkage = b.option(
+        std.builtin.LinkMode,
+        "c_sdl_preferred_linkage",
+        "Prefer building statically or dynamically linked libraries (default: static)",
+    ) orelse .static;
+    const c_sdl_strip = b.option(
+        bool,
+        "c_sdl_strip",
+        "Strip debug symbols (default: varies)",
+    ) orelse (optimize != .Debug);
```

Comment:
```suggestion
    ) orelse (optimize == .ReleaseSmall);
```

https://github.com/ziglang/zig/blob/e96d86064eb81977f254fa8f36481b7d150cb3b6/src/Compilation/Config.zig#L449-L454

https://github.com/ziglang/zig/blob/e96d86064eb81977f254fa8f36481b7d150cb3b6/src/target.zig#L298-L318